docs/reference/configuration.rst

```diff
@@ -184,6 +184,38 @@ CouchDB storage setup based on `doctrine/couchdb-client <https://github.com/doct
 
     $storage = new CouchDbStorage($client);
 
+DynamoDb
+---------
+
+DynamoDb is supported through the `AWS SDK for PHP <https://aws.amazon.com/sdk-for-php/>`_
+Create your tables via the AWS DynamoDb console or using the `PHP based API <http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LowLevelPHPTableOperationsExample.html>`_
+
+.. code-block:: php
+
+    <?php
+
+    $connection_settings = [
+      'profile' => 'default',
+      'region' => \Aws\Common\Enum\Region::AP_SOUTHEAST_2,
+      'key' => 'Your key here, unless you are using IAM roles or .credentials files',
+      'secret' => 'Your secret here, unless you are using IAM roles or .credentials files',
+    ];
+
+    // If you are using local dynamodb for testing, you can set a local url.
+    // $connection_settings['base_url'] = 'http://localhost:8000';
+
+
+    $client = \Aws\DynamoDb\DynamoDbClient::factory($connection_settings);
```

Comment:
You're better off using the `\Aws\Sdk` as a dependency since it has more configuration options that may be relevant. So I'd provide documetation on that instead:

```php
$sdk = new \Aws\Sdk([...]);
$dynamoClient = $sdk->createDynamoDb();
```

You leave the configuration complexity out of this and depend on AWS SDK.